src/ConfigCatClientOptions.ts

```diff
@@ -2,21 +2,31 @@ import { ConfigCatConsoleLogger } from "./ConfigCatLogger";
 import { IConfigCatLogger, IAutoPollOptions, ILazyLoadingOptions, IManualPollOptions, LogLevel } from "./index";
 import COMMON_VERSION from "./Version";
 
+
+/** Restrict the location of your feature flag and setting data within the ConfigCat CDN. */
 export enum DataGovernance {
+    /** Your data will be published to all ConfigCat CDN nodes to guarantee lowest response times. */
     Global = 0,
+    /** Your data will be published to CDN nodes only in the EU. */
     EuOnly = 1
 }
 
 export interface IOptions {
     logger?: IConfigCatLogger;
     requestTimeoutMs?: number;
     baseUrl?: string;
+    /** You can set a base_url if you want to use a proxy server between your application and ConfigCat */
     proxy?: string;
+    /** Set this parameter to restrict the location of your feature flag and setting data within the ConfigCat CDN.
```

Comment:
Set this parameter to be in sync with the Data Governance preference on the Dashboard: https://app.configcat.com/organization/data-governance (Only Organization Admins have access)

Reply:
done